target/maven-archiver/pom.properties

```diff
@@ -1,5 +1,5 @@
 #Generated by Apache Maven
```

Comment:
Could you add the `target` directory to your `.gitignore` file? That way, we don't see these files in the diffs.